### PR TITLE
Css variables

### PR DESCRIPTION
**Pull request by yetieaterxb1**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: src/styles/NewChannel.scss [hunk 1]
```diff
@@ -1,47 +1,45 @@
 .str-chat__new-channel {
```
Comment (DanC5) [inline src/styles/NewChannel.scss:1]: doesn't look like the styles in this file are applied anywhere so we should delete

File: src/styles/Panel.scss [hunk 2]
```diff
@@ -3,9 +3,9 @@
   bottom: calc(100%);
```
Comment (DanC5) [inline src/styles/Panel.scss:3]: doesn't look like the styles in this file are applied anywhere so we should delete

File: src/styles/ReadState.scss [hunk 3]
```diff
@@ -5,5 +5,5 @@
   position: absolute;
```
Comment (DanC5) [inline src/styles/ReadState.scss:5]: doesn't look like the styles in this file are applied anywhere so we should delete

File: src/styles/_variables.scss [hunk 4]
```diff
@@ -1,26 +1,177 @@
...
+  --accent_blue: #005fff;
+  --accent_green: #20e070;
+  --accent_red: #ff3742;
+
+  /* base colors */
+
+  --bg-gradient-end: #f7f7f7;
+  --bg-gradient-start: #fcfcfc;
+  --black: #000000; //keep
+  --blue-alice: #e9f2ff; //keep
```
Comment (DanC5) [inline src/styles/_variables.scss:137]: why some of these say keep

File: src/styles/_variables.scss [hunk 5]
```diff
@@ -1,26 +1,177 @@
...
+  --xxl-font: 26px;
+  --xxxl-font: 32px;
+
+  /* fonts weight */
+  --font-weight-regular: #{$base-font-weight}; //400
+  --font-weight-semi-bold: 600;
+  --font-weight-bold: #{$heavy-font-weight}; //700
+
+  /* colors */
+  --primary-color: #{$secondary-color};
```
Comment (ambaldwin) [inline src/styles/_variables.scss:48]: primary color is set to secondary-color?

Comment (ambaldwin) [reply src/styles/_variables.scss:48]: actually, disregard the above note, and just add a comment to the file here to say why it's being set this way!

**Review by ambaldwin — CHANGES_REQUESTED** [review]

**Review by DanC5 — APPROVED** [review]

**Pull request merged**